docs/recipes/mujoco.md

```diff
@@ -0,0 +1,215 @@
+# Installing and Using MuJoCo
```

Comment:
@lmilechin should we add this at the top? 
---
tags:
 - Engaging
 - SuperCloud
 - Howto Recipes
 - MuJoCo
---